build_all_images.sh

```diff
@@ -54,6 +54,33 @@ function build_batch_image {
 	fi
 }
 
+function build_bundler_builder_image {
+	log_in_to_docker_hub
```

Comment:
@Akos-Kreutz, is it really necessary?

Reply:
Every other script does this when we want to push the image to Docker Hub, so I would say yes. Otherwise the script is not working as a standalone build script.

Reply:
Sorry I just see that this in in the build_all_images.sh, no it's not necessary. I will remove this.

Reply:
Thanks!